src/main/kotlin/net/sacredlabyrinth/phaed/dynmap/simpleclans/tasks/HideWarringClansTask.kt

```diff
@@ -0,0 +1,22 @@
+package net.sacredlabyrinth.phaed.dynmap.simpleclans.tasks
+
+import net.sacredlabyrinth.phaed.dynmap.simpleclans.DynmapSimpleClans
+import org.bukkit.Bukkit
+import org.bukkit.scheduler.BukkitRunnable
+
+const val TASK_INTERVAL: Long = 60 * 20L
```

Comment:
Outside the class definition 🤔

Reply:
## Yeah, it is really strange how Kotlin realize the const initialization.
I was surfing and there are **4 ways**: outside the class (the wrong one? 🤔 ), implement in an object or a companion and using `val`. 

As long as I understand by reading [this article](https://blog.jetbrains.com/kotlin/2013/06/static-constants-in-kotlin/), any `val` is a `const` by default 🤔
> A property in a class object works as well as a *static field* (and even better), but only in Kotlin.

### Playground
Then I decided to test it in [my playground](https://pl.kotl.in/0d601riKe)

### Developers
I watched into the [Kotlin repository](https://github.com/JetBrains/kotlin/blob/6a670dc5f38fc73eb01d754d8f7c158ae0176ceb/libraries/stdlib/unsigned/src/kotlin/UInt.kt#L22), the developers are usually use it in link of `companion/object`. 🤔 
### Compiling?
After all, I decided to see [how it looks in Java](https://paste.md-5.net/zehapulane.java).

`Outside the class` way created the const as it needs, but in a parent class 🤔 
`val` way created the... getter? huh 🤔 Could be the [decompiler](https://extendsclass.com/kotlin-to-java.html) fault 
`companion/object` same as `val`

### Conclusion
I didn't get anything 🤡 , but it seems, `val` is the best choice here.
`Outside the class` way seems that it won't work (the const created in the parent class, not as inner class as I thought).
`Companion/Object` way needs for defining the set of constants. 🤔

Reply:
You put a lot of thought into this. :)